contributors/design-proposals/storage/volume-topology-scheduling.md

```diff
@@ -415,40 +436,31 @@ Kubernetes.
 With all this complexity, the library approach is the most feasible in a single
 release time frame, and aligns better with the current Kubernetes architecture.
 
-#### Downsides/Problems
-
-##### Expanding Scheduler Scope
-This approach expands the role of the Kubernetes default scheduler to also
-schedule PVCs, and not just Pods.  This breaks some fundamental design
-assumptions in the scheduler, primarily that if `Pod.Spec.NodeName`
-is set, then the Pod is scheduled.  Users can set the `NodeName` field directly,
-and controllers like DaemonSets also currently bypass the scheduler.
-
-For this approach to be fully functional, we need to change the scheduling
-criteria to also include unbound PVCs.
+#### Downsides
 
-TODO: details
+##### Unsupported Use Cases
+The following use cases will not be supported for PVCs with a StorageClass with
+VolumeBindingWaitForFirstConsumer:
+* Directly setting Pod.Spec.NodeName
+* DaemonSets
 
-A general extension mechanism is needed to support scheduling and binding other
-objects in the future.
+These two use cases will bypass the default scheduler and thus will not
+trigger PV binding.
 
-##### Impact on Custom Schedulers
-This approach is going to make it harder to run custom schedulers, controllers and
-operators if you use PVCs and PVs. This adds a requirement to schedulers
-that they also need to make the PV binding decision.
+##### Custom Schedulers
+Custom schedulers, controllers and operators that handle pod scheduling and want
+to support this new volume binding mode will also need to handle the volume
+binding decision.
```

Comment:
this makes it clear that the volume binding is a distinct responsibility. combining it into the scheduler in the initial implementation and requiring all other schedulers to implement the same function is a strong indication that the data flow to accomplish this binding is not well separated